source/initial-dns-seedlist-discovery/tests/txt-record-with-overridden-ssl-option.yml

```diff
@@ -0,0 +1,11 @@
+uri: "mongodb+srv://test5.test.build.10gen.cc/?ssl=false"
+seeds:
+    - localhost.test.build.10gen.cc:27017
+hosts:
+    - localhost:27017
+    - localhost:27018
+    - localhost:27019
+options:
+    - authSource: otherDB
+    - replicaSet: repl0
+    - ssl: false
```

Comment:
If `ssl:false` is asserted here, then `ssl:true` should be asserted in all the other tests.

Reply:
I did think about that, but I didn't want to make the assumption that Drivers actually *set* their internal ``ssl`` option to true. But I'll change it as it's more clear and Drivers can address this in their test harness if needed.

Reply:
Fixed